scripts/BilbyCustomFunctions_Reduction.py

```diff
@@ -115,12 +120,93 @@ def strip_NaNs(output_workspace, base_output_name):
 
     return base_output_name
 
+##############################################################################
+# 26 March 2019
+
+
+def output_header(external_mode, used_wl_range, ws_sample, sample_thickness,
+                  sample_transmission, empty_beam_transmission, blocked_beam, sample_mask, transmission_mask):
+    """  Creates header to be recorded into the output file """
+    header = []
+
+    wl_row = 'Velocity selector set wavelength: ' + str(format(float(ws_sample.run().getProperty("wavelength").value), '.3f')) + ' Angstrom'
+    header.append(wl_row)
+
+    if (external_mode):
+        choppers = 'Double choppers pair: ' + str(int(ws_sample.run().getProperty("master1_chopper_id").value)) + ' and ' \
+                                            + str(int(ws_sample.run().getProperty("master2_chopper_id").value))
+        header.append(choppers)
+        frequency = 'Data defining pulse frequency (equal or slower than the Double pair frequency): ' \
+                    + str(format(1e6/float(ws_sample.run().getProperty("period").value), '.2f')) + ' Hz'
+        header.append(frequency)
+        wavelength_range = 'Wavelength range used for the data reduction: ' + str(format(float(used_wl_range[0]), '.2f')) + ' to ' \
+                                                                            + str(format(float(used_wl_range[2]), '.2f')) + ' Angstrom'
+        header.append(wavelength_range)
+        resolution_value = float(used_wl_range[1])
+        if resolution_value < 0:
+            resolution = 'Resolution used for calculation of dQ: ' + str(format((-100 * resolution_value), '.2f')) + '%'
+        else:
+            resolution = 'Resolution taken as wavelength binning;' + '\n' + 'the value is set to ' + \
+                str(format(resolution_value, '.2f')) + '%'  # on linear scale, hence the dQ calculation is meaningless'
+        header.append(resolution)
+    else:
+        resolution = "Nominal resolution: 10%"
+        header.append(resolution)
+
+    L1 = 'L1: ' + str(format(float(ws_sample.run().getProperty("L1").value), '.3f')) + ' m'
```

Comment:
Python variable names should not use upper case letters.

Reply:
> 
> 
> Python variable names should not use upper case letters.

OK, thank you. I'll make all corrections and update the pull request.
Anna